### PR TITLE
JWT before live777 handler()

### PR DESCRIPTION
**Pull request by WinterJack002**
set whipclient publish token=user.token for /wh[i,e]p/{uuid} api.go handler

### CODE REVIEW

File: webapp/components/use/whip.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useSyncExternalStore } from 'react'
 import { event, Context, Data } from './whxp'
-import { Stream, StreamState } from '../../lib/api'
+import { Stream, StreamState, token} from '../../lib/api'
```
Comment (a-wing) [inline webapp/components/use/whip.ts:3]: `, token}` => `, token }`

File: webapp/components/use/whip.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import {
   asyncGetVideoStream,
 } from '../../lib/device'
 
+
```
Comment (a-wing) [inline webapp/components/use/whip.ts:12]: Do not add useless blank lines

**Comment by WinterJack002** [comment]
截图 2024-09-21 09-26-49
Is the reason for this failure caused by this code modification?

**Comment by a-wing** [comment]
> Is the reason for this failure caused by this code modification?

https://github.com/WinterJack002/woom/actions/runs/12021482871/job/33512071837#step:4:7

**Pull request merged**